doc/manual/meson.build

```diff
@@ -4,6 +4,8 @@ project('nix-manual',
   license : 'LGPL-2.1-or-later',
 )
 
+fs = import('fs')
```

Comment:
did you import this but not end up using it?

Reply:
Yes indeed, fixed.